src/generator.ts

```diff
@@ -79,10 +79,14 @@ export class Generator {
     const parameters = parameter.split(',');
     for (let param of parameters) {
       // remove double quote
-      param = param.substring(1, param.length - 1);
+      if (param[0] === '"' && param[param.length - 1] === '"') {
```

Comment:
is parm.lenght guaranteed to be >=1?

Reply:
It does not matter, it won't crash in JavaScript (will return `undefined`).